packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_dex_poolswap.test.ts

```diff
@@ -186,10 +180,7 @@ describe('dex.poolswap()', () => {
       fromAmount: new BigNumber('2'),
       toScript: script,
       toTokenId: pairs.CAT.tokenB,
-      maxPrice: {
-        integer: new BigNumber('0'),
-        fraction: new BigNumber('0.3')
-      }
+      maxPrice: new BigNumber('0.00000003')
```

Comment:
Do test the behavior that maxPrice works as expected in Txn Builder where you test max price should fail right on the delta values of +/- 0.00000001